red-hat-storage/ocs-operator
# noobaa autoscaler changes, new properties added

**Explain the changes**
Integrate noobaa with Prometheus adaptor for custom metrics autoscaling
This PR will make HPAV2 the default HPA for noobaa installed through OCS,
Autoscaler properties `AutoscalerType` and `PrometheusNamespace` added to Noobaa CR with value `hpav2` and hardcoded Prometheus namespace in openshift; `openshift-monitoring`

**Issues: Fixed #xxx / Gap #xxx**
https://bugzilla.redhat.com/show_bug.cgi?id=2217904

**Testing Instructions:**
Install ODF in cluster
check for the newly created HPA2 autoscaler
update noobaa endpoint min and max pod count in Noobaa CR
Verify new endpoint pod is created/deleted
